javascript/javascript1/week3/note-pad.js

```diff
@@ -0,0 +1,43 @@
+const notes = [];
+
+function saveNote(content, id) {
+    //Code goes here here
+    notes.push({ content: content, id: id });
```

Comment:
when the name of object property is the same of the variable name. you can do it shorthand
```js
notes.push({ content, id });
```